openapi/spec/openapi.json

```diff
@@ -2611,6 +2645,33 @@
                 }
               }
             }
+          },
+          "failures": {
+            "type": "object",
+            "items": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "users": {
+                  "type": "array",
+                  "description": "The array of user objects that could not be imported successfully",
+                  "properties": {
```

Comment:
`type: array` and `properties` are not compatible. Did you mean `items`?

Reply:
Updated it.